test/create-fetchival.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const tape = require('tape')
+const createFetchival = require('../create-fetchival')
+const fetch = require('../fetch')
+
+tape('createFetchival', (t) => {
+  t.test('uses injected fetch to fetch json', async (t) => {
+    const fetchival = createFetchival({ fetch })
+
+    const res = await fetchival('https://jsonplaceholder.typicode.com')('posts').get()
+
+    t.equals(res.length, 100)
```

Comment:
how does this test that it used the injected fetch?

Reply:
implicitly because the factory doesn't provide a default for fetch, but I agree it's not ideal. Made it clearer in https://github.com/ExodusMovement/fetch/pull/8/commits/619012b940d5d5fe07600ba5276a74312dd0a686